libkbfs/kbfs_ops_test.go

```diff
@@ -4479,7 +4479,8 @@ func TestKBFSOpsBackgroundFlush(t *testing.T) {
 
 	// start the background flusher
 	ops := getOps(config, rootNode.GetFolderBranch().Tlf)
-	go ops.backgroundFlusher(1 * time.Millisecond)
+	config.SetBGFlushPeriod(1 * time.Millisecond)
```

Comment:
Probably overkill, but maybe we should be using a fake clock here. I won't push on whatever you decide though.